readthedocs/rtd_tests/tests/test_docsitalia.py

```diff
@@ -537,3 +537,45 @@ def test_on_webhook_github_signal_ignores_invalid_branches(self):
     def test_we_use_docsitalia_builder_conf_template(self):
         template = get_template('doc_builder/conf.py.tmpl')
         self.assertIn('readthedocs/templates/doc_builder/conf.py.tmpl', template.origin.name)
+
+    def test_projects_by_tag_api(self):
+        project = Project.objects.create(
+            name='my project',
+            slug='myprojectslug',
+            repo='https://github.com/testorg/myrepourl.git'
+        )
+        data = {
+            'ref': 'master'
+        }
+        with requests_mock.Mocker() as rm:
+            rm.get(
+                'https://raw.githubusercontent.com/testorg/'
+                'myrepourl/master/document_settings.yml',
+                text=DOCUMENT_METADATA)
+            webhook_github.send(Project, project=project, data=data, event='push')
+        project.refresh_from_db()
+        response = self.client.get(reverse('projects-by-tag-list'), {'tags': 'amazing document'})
+        self.assertEqual(len(response.data['results']), 1)
+        self.assertEqual(response.status_code, 200)
+
+    def test_projects_by_tag_api_manual_add(self):
+        project = Project.objects.create(
+            name='my project',
+            slug='myprojectslug',
+            repo='https://github.com/testorg/myrepourl.git'
+        )
+        project.tags.add('lorem', 'ipsum')
+        response = self.client.get(reverse('projects-by-tag-list'), {'tags': 'lorem, sicut'})
+        self.assertEqual(len(response.data['results']), 1)
+        self.assertEqual(response.status_code, 200)
+
+    def test_projects_by_tag_api_fail(self):
```

Comment:
Looks like test_projects_by_tag_returns_only_data_that_matches_tags or similar.
You are missing a test with no tags it returns an empty queryset